rsconnect_jupyter/static/connect.js

```diff
@@ -1033,6 +1033,11 @@ define([
         $subdirsBox.prop('checked', false);
       }
       $subdirsBox.parent().toggleClass('rsc-text-light', !canIncludeSubdirs);
+
+      // Disable hide code cells options in Voila mode, since Voila always hides them.
```

Comment:
The code cell boxes are already greyed out when Voila mode is selected. Is there more that should be done to complete #348 ?

Reply:
We should review the docs on hiding code cells to see if any updates are needed.